upgrade/scripts/upgrade/prerequisites.sh

```diff
@@ -1145,6 +1120,34 @@ else
     echo "====> ${state_name} has been completed" | tee -a "${LOG_FILE}"
 fi
 
+state_name="VALIDATE_SPIRE_POSTGRES_HEALTH"
+state_recorded=$(is_state_recorded "${state_name}" "$(hostname)")
+if [[ ${state_recorded} == "0" ]]; then
+  echo "====> ${state_name} ..." | tee -a "${LOG_FILE}"
+  i=1
+  max_checks=40
+  wait_time=15
+  healthy="false"
+  while [[ ${i} -le ${max_checks} ]]; do
+    if [[ "$(kubectl get postgresql -n spire spire-postgres -ojsonpath='{.status.PostgresClusterStatus}')" == "Running" ]]; then
+      echo "spire-postgres is healthy"
+      healthy="true"
+      break
+    fi
+    echo "Waiting for postgres operator to consider spire-postgres healthy (${i}/${max_checks})"
+    sleep ${wait_time}
+    i=$(( i+1 ))
+  done
+    
+  if [[ ${healthy} == "false" ]]; then
+    echo "ERROR: spire-postgres is not healthy after $(( max_checks * wait_time )) seconds"
+    exit 1
+  fi
+    record_state "${state_name}" "$(hostname)" | tee -a "${LOG_FILE}"
```

Comment:
Indentation is off on this line.  Should like up with the line before it.

Reply:
I ran shfmt on it. I'm hitting too many issues trying to do this with my tools turned off.